README.md

```diff
@@ -0,0 +1 @@
+# Helo this is a RP.
```

Comment:
✏️ **Typo**: Possible typo: 'Helo'. Did you mean 'help'?

```suggestion
help
```

✏️ **Typo**: Possible typo: 'Helo'. Did you mean 'help'?

```suggestion
help
```